docs/components/movement-sensor/gps/dual-gps-rtk.md

```diff
@@ -0,0 +1,108 @@
+images: ["/icons/components/imu.svg"]
+# SMEs: Rand
+---
+
+The `dual-gps-rtk` model of movement sensor calculates compass heading from two gps movement sensors.
+In addition to [`CompassHeading()`](/components/movement-sensor/#getcompassheading), this model provides data for [`GetPosition()`](/components/movement-sensor/#getposition).
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and click **Create component**.
+Select the `movement-sensor` type, then select the `dual-gps-rtk` model.
+Enter a name for your movement sensor and click **Create**.
+
+{{< imgproc src="/components/movement-sensor/dual-gps-rtk-builder.png" alt="Creation of a `dual-gps-rtk` movement sensor in the Viam app config builder." resize="600x" >}}
+
+Copy and paste the following attribute template into your movement sensor's **Attributes** box.
+Then remove and fill in the attributes as applicable to your movement sensor, according to the table below.
+
+{{< tabs >}}
+{{% tab name="Attributes template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "first_gps": "<name-of-your-first-gps-movement-sensor>",
+  "second_gps": "<name-of-your-second-gps-movement-sensor>",
+  "offset_degrees": <int>
+}
+```
+
+{{% /tab %}}
+{{% tab name="Attributes example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "first_gps": "nmea-1",
+  "second_gps": "nmea-2",
+  "offset_degrees": 90
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "your-dual-gps-rtk",
+      "model": "dual-gps-rtk",
+      "type": "movement_sensor",
+      "namespace": "rdk",
+      "attributes": {
+        "first_gps": "<name-of-your-first-gps-movement-sensor>",
+        "second_gps": "<name-of-your-second-gps-movement-sensor>",
+        "offset_degrees": <int>
+      },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "your-dual-gps-rtk",
+      "model": "dual-gps-rtk",
+      "type": "movement_sensor",
+      "namespace": "rdk",
+      "attributes": {
+        "first_gps": "nmea-1",
+        "second_gps": "nmea-2",
+        "offset_degrees": 90
+      },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for a `dual-gps-rtk` movement sensor:
+
+<!-- prettier-ignore -->
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `first_gps` | int | **Required** | The name you have configured for the first movement sensor you want to combine the measurements from. Must be a `GPS` model. |
```

Comment:
Technically we're only checking if the movement sensor is position reporting in the code, but I doubt this model will be useful for non-GPS position reporting sensors such as wheeled odometry, SLAM algorithms or optical trackers.

Reply:
Should I specify position reporting then, or leave as is?

Reply:
Leaving as is is more useful.

Reply:
Ok, thanks!